Control/Monad/Combinators.hs

```diff
@@ -162,6 +163,8 @@ many p = go id
 -- | @'manyTill' p end@ applies parser @p@ /zero/ or more times until parser
 -- @end@ succeeds. Returns the list of values returned by @p@.
 --
+-- Notice that @end@ result is consumed and lost. Use @manyTill_@ if you wish to keep it.
```

Comment:
You can use `'thisSyntax'` to greate hyperlinks in Haddocks.

Reply:
Done so.